src/main/java/news/caughtup/model/Article.java

```diff
@@ -19,18 +23,29 @@ public String getTitle() {
     public void setTitle(String title) {
         this.title = title;
     }
-    public String getDate() {
+    
+    public int getResourceId() {
+        return resourceId;
+    }
+
+    public void setResourceId(int resourceId) {
```

Comment:
You don't wanna change the resourceId after creating the object. It might lead to inconsistencies.